addons/l10n_pt/models/chart_template.py

```diff
@@ -0,0 +1,40 @@
+from odoo import models, _, api
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = "account.chart.template"
+
+    def _get_account_vals(self, company, account_template, code_acc, tax_template_ref):
+        vals = super()._get_account_vals(company, account_template, code_acc, tax_template_ref)
+        if company.account_fiscal_country_id.code != 'PT':
+            return vals
+        vals['l10n_pt_taxonomy_code'] = account_template.l10n_pt_taxonomy_code
+        return vals
+
+    @api.model
+    def _get_default_bank_journals_data(self):
+        if self.env.company.account_fiscal_country_id.code != 'PT':
+            return super()._get_default_bank_journals_data()
+        return [{
+            'name': _('Cash'),
+            'type': 'cash',
+            'l10n_pt_taxonomy_code': 1,
+        }, {
+            'name': _('Bank'),
+            'type': 'bank',
+            'l10n_pt_taxonomy_code': 2,
+        }]
```

Comment:
why not use the super call?
```suggestion
    @api.model
    def _get_default_bank_journals_data(self):
        journal_values_list = super()._get_default_bank_journals_data()
        if self.env.company.account_fiscal_country_id.code == 'PT':
            for journal_values in journal_values_list:
                if journal_values['account_type'] == 'cash':
                    journal_values['l10n_pt_taxonomy_code'] = 1
                elif journal_values['account_type'] == 'bank':
                    journal_values['l10n_pt_taxonomy_code'] = 2
        return journal_values_list
```